README.md

```diff
@@ -1,2 +1,27 @@
-# quick_search-archives_space_searcher
 ArchivesSpace searcher for NCSU Quick Search
+=======
+UMD ArchivesSpace searcher for NCSU Quick Search
+
+## Installation
+Include the searcher gem in your Gemfile:
+
+```
+gem 'quick_search-library_website_searcher'
+
+```
+And then execute:
+```bash
+$ bundle install
+```
+
+## Configuration
+
+The ArchivesSpace searcher requires configuration, such as specific URL to
+use in retrieving search results and CSS selectors to choose results.
+To set the configuration, create a "config/searchers/" directory in your 
+application (the "searchers" subdirectory may need to be created), 
+and copy the "config/archives_space_config.yml" file
+in this gem into it. Follow the instructions in the file to configure the
+searcher.
+
```

Comment:
For consistency with the other searchers, please indicate the "render_module" stanza to use in the QuickSearch application. See https://github.com/umd-lib/quick_search-swiftype_searcher